BlockstackInstance+ObjC.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  BlockstackInstance+ObjC.swift
+//  Blockstack
+//
+//  Created by Michal Ciurus on 09/05/2018.
+//
+
+import Foundation
+
+@objc public class ProfileObject: NSObject {
```

Comment:
Swift structs aren't supported in Objective-C so they need to be translated to classes.

I'd suggest hiding *all* of Blockstack API methods behind a façade (like in my [proposal PR[(https://github.com/blockstack/blockstack-ios/pull/2)). This would simplify Objective-C support as only one file would have to be changed (the façade) and internal implementation of Blockstack wouldn't have any `NSObject` or `@objc` keywords spoiling it.